commons-numbers-combinatorics/src/main/java/org/apache/commons/numbers/combinatorics/LogFactorial.java

```diff
@@ -41,7 +41,7 @@
      *
      * @param numValues Number of values of the function to compute.
      * @param cache Cached values.
-     * @throw IllegalArgumentException if {@code n < 0}.
+     * @throw CombinatoricsException if {@code n < 0}.
```

Comment:
You have left this `CombinatoricsException` exposed in the API.

Reply:
Changed
TY